README.md

```diff
@@ -119,6 +119,37 @@ You will have a file called flipper.contract. You can use this directly in
 the [Polkadot UI](https://substrate.dev/substrate-contracts-workshop/#/0/deploy-contract),
 as if your smart contract was written using ink!.
 
+## Roadmap
+
+Solang has a long way to go in order to achieve full compatibility for the blockchains we target.
+Here is a brief description of what we envision for the next versions.
+
+### V0.2
+
+| Milestone | Status |
+| --------- | ------ |
+| Specify values as "1 sol" and "1e9 lamports" | Not started |
+| Solana SPL tokens compatibility | In progess |
+
+### V0.3
+
+| Milestone | Status |
+| --------- | ------ |
+| Call Rust contracts from Solidity | Not started |
+| Parse and resolve inline assembly | In progress |
+
```

Comment:
Any reason for not including the other versions?

Reply:
The readme would get too long, and I don't think it should.

Reply:
ok